src/current/_includes/v22.2/orchestration/kubernetes-stop-cluster.md

```diff
@@ -7,37 +7,41 @@ To shut down the CockroachDB cluster:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ kubectl delete -f example.yaml
+    kubectl delete -f example.yaml
     ~~~
 
 1. Remove the Operator:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ kubectl delete -f https://raw.githubusercontent.com/cockroachdb/cockroach-operator/v{{ latest_operator_version }}/install/operator.yaml
+    kubectl delete -f https://raw.githubusercontent.com/cockroachdb/cockroach-operator/v{{ latest_operator_version }}/install/operator.yaml
     ~~~
 
-    This will delete the CockroachDB cluster being run by the Operator. It will *not* delete the persistent volumes that were attached to the pods. 
+    This will delete the CockroachDB cluster being run by the Operator. It intentionally does _not_ delete:
 
-    {{site.data.alerts.callout_danger}}
-    If you want to delete the persistent volumes and free up the storage used by CockroachDB, be sure you have a backup copy of your data. Data **cannot** be recovered once the persistent volumes are deleted. For more information, see the [Kubernetes documentation](https://kubernetes.io/docs/tasks/run-application/delete-stateful-set/#persistent-volumes).
-    {{site.data.alerts.end}}
+    - The persistent volumes that were attached to the pods, to avoid the risk of unintentional data loss. Before deleting a cluster's persistent volumes, be sure to back them up. For more information, refer to [Delete a Cluster's Persistent Volume](#delete-a-clusters-persistent-volumes) in the Kubernetes project's documentation.
```

Comment:
```suggestion
    - The persistent volumes that were attached to the pods, to avoid the risk of data loss. Before deleting a cluster's persistent volumes, be sure to back them up. For more information, refer to [Delete a Cluster's Persistent Volume](#delete-a-clusters-persistent-volumes) in the Kubernetes project's documentation.
```

Reply:
Maybe I'm missing something in the meaning here, but it seems like "unintentional" is redundant here.